scenarios/kubernetes_e2e_test.py

```diff
@@ -393,9 +392,13 @@ def test_default_tag(self):
         self.assertIsNotNone(match)
         url = 'https://gcr.io/v2/%s/manifests/%s' % (match.group(1),
                                                      match.group(2))
-        data = json.loads(urllib.urlopen(url).read())
+        print url
```

Comment:
nit: remove this?

Reply:
Yup will do - sorry - forgot to remove my debugging code!